javascript/src/actions/variables.js

```diff
@@ -105,3 +106,10 @@ export const requestTransfer = variable => {
         variable
     }
 }
+
+export const receiveVariablesRegion = region => {
```

Comment:
`receive` probably isn't the best term, since we're typically using that for data returned from an async call (there don't seem to be any API calls involved here).

Instead, how about `setVariablesRegion` / `SET_VARIABLES_REGION`?